src/components/landingPage/_LandingPageSidebar.svelte

```diff
@@ -0,0 +1,74 @@
+				/>
+			</svg>
+		</svelte:fragment>
+		<span>Features</span>
+	</AppRailTile>
+	<AppRailTile bind:group={tile} name="reviewTile" target="reviewBox" value={1}>
+		<svelte:fragment slot="lead">
+			<svg
+				xmlns="http://www.w3.org/2000/svg"
+				viewBox="0 0 24 24"
+				fill="currentColor"
+				class="w-6 h-6 mx-auto"
+			>
+				<path
+					fill-rule="evenodd"
+					d="M9 4.5a.75.75 0 01.721.544l.813 2.846a3.75 3.75 0 002.576 2.576l2.846.813a.75.75 0 010 1.442l-2.846.813a3.75 3.75 0 00-2.576 2.576l-.813 2.846a.75.75 0 01-1.442 0l-.813-2.846a3.75 3.75 0 00-2.576-2.576l-2.846-.813a.75.75 0 010-1.442l2.846-.813A3.75 3.75 0 007.466 7.89l.813-2.846A.75.75 0 019 4.5zM18 1.5a.75.75 0 01.728.568l.258 1.036c.236.94.97 1.674 1.91 1.91l1.036.258a.75.75 0 010 1.456l-1.036.258c-.94.236-1.674.97-1.91 1.91l-.258 1.036a.75.75 0 01-1.456 0l-.258-1.036a2.625 2.625 0 00-1.91-1.91l-1.036-.258a.75.75 0 010-1.456l1.036-.258a2.625 2.625 0 001.91-1.91l.258-1.036A.75.75 0 0118 1.5zM16.5 15a.75.75 0 01.712.513l.394 1.183c.15.447.5.799.948.948l1.183.395a.75.75 0 010 1.422l-1.183.395c-.447.15-.799.5-.948.948l-.395 1.183a.75.75 0 01-1.422 0l-.395-1.183a1.5 1.5 0 00-.948-.948l-1.183-.395a.75.75 0 010-1.422l1.183-.395c.447-.15.799-.5.948-.948l.395-1.183A.75.75 0 0116.5 15z"
+					clip-rule="evenodd"
+				/>
+			</svg>
+		</svelte:fragment>
+		<span>Reviews</span>
+	</AppRailTile>
+	<AppRailTile bind:group={tile} name="contactTile" target="contactBox" value={2}>
+		<svelte:fragment slot="lead">
+			<svg
+				xmlns="http://www.w3.org/2000/svg"
+				viewBox="0 0 24 24"
+				fill="currentColor"
+				class="w-6 h-6 mx-auto"
+			>
+				<path
+					fill-rule="evenodd"
+					d="M4.848 2.771A49.144 49.144 0 0112 2.25c2.43 0 4.817.178 7.152.52 1.978.292 3.348 2.024 3.348 3.97v6.02c0 1.946-1.37 3.678-3.348 3.97a48.901 48.901 0 01-3.476.383.39.39 0 00-.297.17l-2.755 4.133a.75.75 0 01-1.248 0l-2.755-4.133a.39.39 0 00-.297-.17 48.9 48.9 0 01-3.476-.384c-1.978-.29-3.348-2.024-3.348-3.97V6.741c0-1.946 1.37-3.68 3.348-3.97zM6.75 8.25a.75.75 0 01.75-.75h9a.75.75 0 010 1.5h-9a.75.75 0 01-.75-.75zm.75 2.25a.75.75 0 000 1.5H12a.75.75 0 000-1.5H7.5z"
+					clip-rule="evenodd"
+				/>
+			</svg>
+		</svelte:fragment>
```

Comment:
kann man die einzelnen Fragments nicht auch jeweils in Komponenten auslagern?

Reply:
bitte nicht, ich bin wirklich kein fan von unnötigem aufbrechen von klassen. das hätte null gewinn, da wir die komponenten nicht wiederverwenden und es würde einfach nur ein ctrl + leftclick mehr werden :x